packages/device_info_plus/device_info_plus/macos/device_info_plus.podspec

```diff
@@ -1,21 +1,22 @@
 #
-# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint device_info_plus_macos.podspec' to validate before publishing.
 #
 Pod::Spec.new do |s|
   s.name             = 'device_info_plus'
   s.version          = '0.0.1'
-  s.summary          = 'No-op implementation of the macos device_info_plus to avoid build issues on macos'
+  s.summary          = 'A new flutter plugin project.'
   s.description      = <<-DESC
- No-op implementation of the device_info_plus plugin to avoid build issues on macos.
-https://github.com/flutter/flutter/issues/46618
-                          DESC
-  s.homepage         = 'https://github.com/fluttercommunity/plus_plugins/tree/master/packages/device_info_plus'
+A new flutter plugin project.
+                       DESC
+  s.homepage         = 'http://example.com'
   s.license          = { :file => '../LICENSE' }
-  s.author           = { 'Flutter Community' => 'authors@fluttercommunity.dev' }
+  s.author           = { 'Your Company' => 'email@example.com' }
```

Comment:
small nit, would be good to keep the `'Flutter Community' => 'authors@fluttercommunity.dev'`

Reply:
Didn't see this, thanks!